tailwind.theme.conf.cjs

```diff
@@ -1 +1 @@
nts--tabs-colors-bg-secondary)","secondary-accent":"var(--components--tabs-colors-bg-secondary-accent)","secondary-subtle":"var(--components--tabs-colors-bg-secondary-subtle)","body":"var(--components--tabs-colors-bg-body)"},"bd":{"primary":"var(--components--tabs-colors-bd-primary)","secondary":"var(--components--tabs-colors-bd-secondary)"},"fg":{"default":"var(--components--tabs-colors-fg-default)","accent":"var(--components--tabs-colors-fg-accent)","cta":"var(--components--tabs-colors-fg-cta)","cta-inverse":"var(--components--tabs-colors-fg-cta-inverse)"}},"font-family":{"open-sans":"var(--components--tabs-font-family-open-sans)"},"font-size":{"xs":"var(--components--tabs-font-size-xs)","s":"var(--components--tabs-font-size-s)","m":"var(--components--tabs-font-size-m)","l":"var(--components--tabs-font-size-l)","xl":"var(--components--tabs-font-size-xl)","2xl":"var(--components--tabs-font-size-2xl)","3xl":"var(--components--tabs-font-size-3xl)"},"font-weight":{"regular":"var(--components--tabs-font-weight-regular)","semibold":"var(--components--tabs-font-weight-semibold)","bold":"var(--components--tabs-font-weight-bold)"},"line-height":{"xs":"var(--components--tabs-line-height-xs)","s":"var(--components--tabs-line-height-s)","m":"var(--components--tabs-line-height-m)","l":"var(--components--tabs-line-height-l)","xl":"var(--components--tabs-line-height-xl)","2xl":"var(--components--tabs-line-height-2xl)","3xl":"var(--components--tabs-line-height-3xl)"},"width":{"page-min":"var(--components--tabs-width-page-min)","page-max":"var(--components--tabs-width-page-max)"},"border-radius":{"none":"var(--components--tabs-border-radius-none)","xs":"var(--components--tabs-border-radius-xs)","s":"var(--components--tabs-border-radius-s)","m":"var(--components--tabs-border-radius-m)","l":"var(--components--tabs-border-radius-l)","full":"var(--components--tabs-border-radius-full)"},"box-shadow":{"none":"var(--components--tabs-box-shadow-none)","normal":"var(--components--tabs-box-shadow-normal)","highlighted":"var(--components--tabs-box-shadow-highlighted)"},"spacing":{"auto":"var(--components--tabs-spacing-auto)","none":"var(--components--tabs-spacing-none)","xs":"var(--components--tabs-spacing-xs)","s":"var(--components--tabs-spacing-s)","m":"var(--components--tabs-spacing-m)","l":"var(--components--tabs-spacing-l)","xl":"var(--components--tabs-spacing-xl)","xxl":"var(--components--tabs-spacing-xxl)"},"z-index":{"hide":"var(--components--tabs-z-index-hide)","base":"var(--components--tabs-z-index-base)","raised":"var(--components--tabs-z-index-raised)","dropdown":"var(--components--tabs-z-index-dropdown)","sticky":"var(--components--tabs-z-index-sticky)","overlay":"var(--components--tabs-z-index-overlay)","modal":"var(--components--tabs-z-index-modal)","popover":"var(--components--tabs-z-index-popover)","skip-link":"var(--components--tabs-z-index-skip-link)","toast":"var(--components--tabs-z-index-toast)","tooltip":"var(--components--tabs-z-index-tooltip)"}}}}
```

Comment:
Maybe you should re-generate this file, as it is a little bit bloated following our experiments 😄

Reply:
Fixed :)